vra/data_source_block_device.go

```diff
@@ -15,83 +15,109 @@ func dataSourceBlockDevice() *schema.Resource {
 		Read: dataSourceBlockDeviceRead,
 
 		Schema: map[string]*schema.Schema{
-			"id": {
-				Type:     schema.TypeString,
-				Optional: true,
-				Computed: true,
-			},
-			"filter": {
-				Type:     schema.TypeString,
-				Optional: true,
-			},
 			"capacity_in_gb": {
-				Type:     schema.TypeInt,
-				Computed: true,
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: "Capacity of the block device in GB.",
 			},
 			"cloud_account_ids": {
 				Type:     schema.TypeList,
 				Computed: true,
 				Elem: &schema.Schema{
 					Type: schema.TypeString,
 				},
+				Description: "Id of the cloud account this storage profile belongs to.",
 			},
 			"created_at": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Date when the entity was created. The date is in ISO 8601 and UTC.",
 			},
 			"custom_properties": {
-				Type:     schema.TypeMap,
-				Computed: true,
+				Type:        schema.TypeMap,
+				Computed:    true,
+				Description: "\"Additional custom properties that may be used to extend the block device.",
 			},
 			"deployment_id": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The id of the deployment that is associated with this resource.",
 			},
 			"description": {
-				Type:     schema.TypeString,
-				Optional: true,
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "A human-friendly description.",
+			},
+			"expand_snapshots": {
```

Comment:
Minor suggestion: It will be easier to read the code if all required and optional properties are before the computed properties sorted in their own categories

Reply:
Done.